java/libs/src/main/java/org/physical_web/collection/PhysicalWebCollection.java

```diff
@@ -298,6 +298,22 @@ public void setPwsEndpoint(String pwsEndpoint) {
     mPwsClient.setPwsEndpoint(pwsEndpoint);
   }
 
+  /**
+   * Set the API version the PWS endpoint uses.
+   * @param pwsApiVersion The new PWS endpoint.
+   */
+  public void setPwsApiVersion(int pwsApiVersion) {
```

Comment:
Same comments here as in the PwsClient...I think it would be good just to make these params to one method.